File: Modul 6/SOAL 1/C/PRAK601.c
#include <stdio.h>

int main() {
    int baris, kolom, elemen;
    scanf("%d %d", &baris, &kolom);
    
    int matriks[baris][kolom];

    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &elemen);
            matriks[i][j] = elemen;
        }
    }
    printf("\n");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Modul 6/SOAL 1/PYTHON/PRAK601.py
baris, kolom = map(int, input().split())
elemen = list(map(int, input().split()))

matriks = []

for i in range(baris):
    baris_matriks = []
    for j in range(kolom):
        baris_matriks.append(elemen[i * kolom + j])
    matriks.append(baris_matriks)

print()

for i in range(baris):
    for j in range(kolom):
        print(matriks[i][j], end=" ")
    print()